helm-chart/ray-cluster/values.yaml

```diff
@@ -52,6 +52,8 @@ head:
 
 
 worker:
+  ## If you want to disable the default worker group
+  # disabled: true
```

Comment:
As below, I think it would be better to have `enabled` , with the opposite meaning.

Reply:
Nevermind!